config.yaml

```diff
@@ -38,6 +38,16 @@ mongo_username: ""
 # Overwrite with environment variable: DEVICECONFIG_MONGO_PASSWORD
 mongo_password: ""
 
+# Inventory service URI
+# Defaults to: "http://mender-inventory:8080"
+# Overwrite with environment variable: DEVICECONFIG_INVENTORY_URI
+DEVICECONFIG_INVENTORY_URI: http://mender-inventory:8080
+
+# Inventory timeout
+# Defaults to: 10
+# Overwrite with environment variable: DEVICECONFIG_INVENTORY_TIMEOUT
+DEVICECONFIG_INVENTORY_TIMEOUT: 10
+
```

Comment:
@imortkz may I ask you why you need these settings here?
The key are `inventory_uri` and `inventory_timeout`, as defined here: https://github.com/mendersoftware/deviceconfig/blob/master/config/config.go#L59

The values you are setting are the defaults, thus even in the case we update the keys, these configuration settings are no-ops.

Can you please help me understand what you are fixing with this PR?